manifests/b/BandicamCompany/Bandicam/5.1.1.1837/BandicamCompany.Bandicam.installer.yaml

```diff
@@ -1,10 +0,0 @@
-# Automatically updated by the winget bot at 2021/Jun/01
-PackageIdentifier: BandicamCompany.Bandicam
-PackageVersion: 5.1.1.1837
-Installers:
-- Architecture: x86
-  InstallerType: nullsoft
-  InstallerUrl: https://dl.bandicam.com/bdcamsetup.exe

```

Comment:
Rather than removing the old version, why not change the URL to the historic URL?
`InstallerUrl: https://dl.bandicam.com/old/BDCAMSETUP_ENG_5_1_1_1837.EXE`

Reply:
Thanks!